tests/swarm/test_reliable_delivery.py

```
@@ -0,0 +1,573 @@
+        # NACK for non-existent sequence
+        delivery.handle_nack(999, reason="congestion")
+        
+        # Should not crash
+        assert delivery.get_pending_count() == 0
+    
+    def test_max_retries_exceeded(self):
+        """Test behavior when max retries exceeded."""
+        config, agent_id = create_config()
+        bus = create_bus(config)
+        
+        delivery = ReliableDelivery(bus, agent_id)
```

Ruff (0.14.10)

Local variable `delivery` is assigned to but never used

Remove assignment to unused variable `delivery`

(F841)